R/utilities.R

```diff
@@ -913,3 +913,31 @@ track_recent_covid_increases <- function(
   }
   return(out)
 } 
+
+update_fac_outbreaks_sheet <- function() {
+  scrape_df <- behindbarstools::read_scrape_data(all_dates = TRUE)
+  outbreaks_sheet_loc = "1I7oubSBZT1GnDL30f4jHzIQwQGso5RulrrBUgxFfRAM"
```

Comment:
So when I was testing this locally, I expected that if I changed the value of this variable, I could run the function without overwriting the google sheet that we're currently using. But I accidentally wrote to the actual sheet because I didn't realized that the actual changing of the values happened in `track_recent_covid_increases` where the same sheet value is hard-coded.